src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -1764,6 +1768,30 @@ SendQueryInSingleRowMode(MultiConnection *connection, char *query,
 void
 ExtractParametersFromParamListInfo(ParamListInfo paramListInfo, Oid **parameterTypes,
 								   const char ***parameterValues)
+{
+	ExtractParametersFromParamListInfoInternal(paramListInfo, parameterTypes,
+											   parameterValues, false);
+}
+
+
+/*
```

Comment:
The comment is the same with `ExtractParametersFromParamListInfo `, I think it'd be good if we can have different comments for the functions.

Also, the function name(s) sounds a little bit hard to understand, what about we rename to so that it is pretty clear to the readers : `ExtractParametersFromParamListInfoLocal ` > `ExtractParametersForLocalExecution`
`ExtractParametersFromParamListInfo ` -> `ExtractParametersForRemoteExecution `